php/class-terminus-command.php

```diff
@@ -153,9 +153,14 @@ public static function request($realm, $uuid, $path = FALSE, $method = 'GET', $o
       );
       $cache->put_data($cachekey, $data);
       return $data;
-    } catch( Exception $e ) {
+    } catch( Guzzle\Http\Exception\BadResponseException $e ) {
       $response = $e->getResponse();
       \Terminus::error("%s", $response->getBody(TRUE) );
+    } catch( Guzzle\Http\Exception\BadResponseException $e ) {
+      $request = $e->getRequest();
+      \Terminus::error("Request %s had failed: %s", (string)$request, $e->getMessage() );
```

Comment:
I understand why you did it but I think there's a typo because the exception class referred to in lines 156 and 159 seem to be the same.